app/components/post/post.js

```diff
@@ -80,8 +80,13 @@ class Post extends PureComponent {
     }
 
     componentWillReceiveProps(nextProps) {
-        const {config, license, currentUserId, roles, post} = nextProps;
-        if (nextProps.post !== this.props.post) {
+        if (nextProps.config !== this.props.config ||
+            nextProps.license !== this.props.license ||
+            nextProps.currentUserId !== this.props.currentUserId ||
+            nextProps.post !== this.props.post ||
+            nextProps.roles !== this.props.roles) {
+            const {config, license, currentUserId, roles, post} = nextProps;
```

Comment:
I think we can make this more accurate but 0/5 really

Reply:
What do you mean by more accurate? Do you mean by passing less into the function or using a selector to pass it through props?

Reply:
I mean currentUserId is not really needed as it won't change unless the user logs out at by then the component is unmounted.

for license we only need `IsLicensed` to change and for the config I think there are only 3 props that we check in those functions, but hey that's why I said 0/5